application/backend/src/main/resources/apps/typerefinery/components/graphs/xAxisBarChart/script.js

```diff
@@ -38,7 +38,9 @@ function xAxisBarChartComponent(component, id) {
       drawXAxisBarChart(defaultData, id);
```

Comment:
this file needs tobe refactored into a namespace. what is the meaning of xAxisBarChart? will there be a new component graphs/yAxisBarChart?

Reply:
Yeah, these things will be refactored.
Scripts will be moved to clientLibs as implemented in the ticker component.

Reply:
In this, PR only the ticker component is refactored.